frontend/components.d.ts

```diff
@@ -25,7 +25,6 @@ declare module 'vue' {
     BFormSelect: typeof import('bootstrap-vue-next')['BFormSelect']
     BInputGroup: typeof import('bootstrap-vue-next')['BInputGroup']
     BModal: typeof import('bootstrap-vue-next')['BModal']
-    BPopover: typeof import('bootstrap-vue-next')['BPopover']
```

Comment:
why'd you change sth in the frontend?

Reply:
gute Frage, ich glaub da hat jemand ein Cyberattacke auf mein PC gemacht, weil ich keine Ahnung habe wie das passieren konnte